### ISSUE TITLE
[Oracle] Sessions view: add IO columns

### ISSUE THREAD

**Issue by p91paul** [dbeaver/dbeaver#4030]
Relevant columns are available in system view ``v$sess_io``.

This is the SQL query performed by the Sessions view of TORA, one of your competitors (mostly on Oracle):

    SELECT a.Sid "Id",
       a.Serial# "Serial#",
       a.UserName "Username",
       a.SchemaName "Schema",
       a.Status "Status",
       a.Server "Server",
       a.OsUser "Osuser",
       a.Machine "Machine",
       a.Program "Program",
       a.Type "Type",
       a.Module "Module",
       a.Action "Action",
       a.Client_Info "Client Info",
       b.Block_Gets "Block Gets",
       b.Consistent_Gets "Consistent Gets",
       b.Physical_Reads "Physical Reads",
       b.Block_Changes "Block Changes",
       b.Consistent_Changes "Consistent Changes",
       c.Value*10 "CPU (ms)",
       a.last_call_et "Last SQL",
       a.Process "Client PID",
       e.SPid "Server PID",
       d.sql_text "Current statement",
       a.SQL_Address||':'||a.SQL_Hash_Value " SQL Address",         
       a.Prev_SQL_Addr||':'||a.Prev_Hash_Value " Prev SQl Address", 
       a.SQL_ID as " SQL_ID",                                       
       a.SQL_CHILD_NUMBER as " SQL_CHILD_NUMBER",                   
       a.PREV_SQL_ID as " PREV_SQL_ID",                             
       a.PREV_CHILD_NUMBER as " PREV_CHILD_NUMBER"                  
    FROM v$session a left join v$sess_io b on ( a.sid = b.sid)          
       left join v$sesstat c on ( a.sid = c.sid)                      
       left join v$sql d on (a.sql_address = d.address and            
                             a.sql_hash_value=d.hash_value and        
                             a.sql_child_number = d.child_number)     
       left join v$process e on (a.paddr = e.addr)
    WHERE (c.statistic# = 12 OR c.statistic# IS NULL)
      AND a.Type != 'BACKGROUND' AND a.Status != 'INACTIVE'
    ORDER BY a.Sid

You might want to check out their sessions view, it is VERY featureful. I'm going to open a few tickets about this.

**Comment by serge-rider** [comment]
This would be a good addition. See it in 5.2.1

**Comment by p91paul** [comment]
You closed many issues I opened with 5.2.1; well I tried them, and the session browser is now really awesome. I just want to thank you @serge-rider, and anyone who helps you in any way. Dbeaver is an excellent tool from a technical standpoint, but it is even more excellent for how it is managed, and the fast pace at which not only bugs are fixed, but even new features are added. Not to mention the kindness with which you help anyone asking anything. You deserve to have as much success as possible, I really hope Rider Soft becomes the market leader for database tools. If only I had any kind of influence on my company budget we would be buying dozens of EE licenses...
